### ISSUE TITLE
New Install, Problem Downloading and staying connected

### ISSUE THREAD

**Issue by glenhawx** [openaudible/openaudible#187]
Hi,
I have only used OpenAudible once before and it worked perfectly. I recently reinstalled OpenAudible on the same machine with a fresh install of Linux and I have had some trouble.
First problem was I had not installed Java so it wouldn't run so I installed openjdk-11-jdk and it ran ok but it would only stay connected unless I logged in via the browser every time I opened it.
Also when I attempt to download anything I get the following message...

> WARNING: error downloading Lost in the End: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty

I tried uninstalling openaudible and reinstalling it. Same problem.
I tried uninstalling openjdk-11 and installing openjdk-8; also no change.
I tried googling about Trust Anchors and got nowhere good.

Any advice would be very welcome.
Thanks.

**Comment by openaudible** [comment]
You didn't say what OS you're using.. but this looks like a good fix:

https://stackoverflow.com/questions/6784463/error-trustanchors-parameter-must-be-non-empty

There is a fix for Ubuntu 18.04

**Comment by glenhawx** [comment]
Sorry, I forgot to add I had been installing OpenAudible on ChromeOS in the Crostini linux container which I believe is Debian. I had it up and running previously with no special modifications. Just installed it and it ran and everything worked.
After the reinstall I encountered the problems above.
Thanks for the link, I had found that in my searching but couldn't understand it and wasn't sure it was relevant to my problems.
Thanks for your reply.
I have since installed OpenAudible on my mac instead.
